aspnetcore/fundamentals/aot/native-aot-tutorial.md

```diff
@@ -49,7 +49,7 @@ Create an ASP.NET Core API app that is configured to work with native AOT:
 Run the following commands:
 
 ```dotnetcli
-dotnet new api --aot -o MyFirstAotWebApi && cd MyFirstAotWebApi
+dotnet new api -aot -o MyFirstAotWebApi && cd MyFirstAotWebApi
```

Comment:
I just tried this command and it works prefixed with two hyphens and it doesn't work prefixed by one hyphen. Can you try again to see if -aot works and if it does, what SDK version are you using?

Reply:
OK, thanks Tom, maybe a version difference?.  I tested it on both win and MacOS using CLI with one hyphen and it worked and using two hyphens did not.  I'll test again.  I was using 8.0.100-preview.3.23178.7 and VS 2022 17.7.0 Preview 2.0.  What do you have?

Reply:
Output for CLI below: 1st with two hyphens and 2nd with one hyphen which runs successfully:

```
dotnet new api --aot -o MyFirstAotWebApi
Error: Invalid option(s):
--aot
   '--aot' is not a valid option

For more information, run:
   dotnet new api -h

For details on the exit code, refer to https://aka.ms/templating-exit-codes#127

dotnet new api -aot -o MyFirstAotWebApi
The template "ASP.NET Core API" was created successfully.
```

Reply:
I should have been at ASP.NET Core Runtime 8.0.0-preview.5.23302.2 which went up on the 13th.  I'll go back and test.

Reply:
@tdykstra,  the command did indeed change with the last version.  I changed it back to one hyphen, thanks!  I forgot you added a commit and made the change back locally and committed.  Sorry I bypassed yours, not intentional.  Should be right now.